tests/helpers/erc20IsUp.js

```diff
@@ -0,0 +1,85 @@
+// This file is used to probe the ganache server. It attempts to call
+// balanceOf(0xaaadeed4...) to get the ERC20 token balance.
+// when the standup script has finished, it will return 500 tokens,
+// which is assumed if our call to balanceOf returns a non-zero value.
+const axios = require('axios')
+const url = require('./url')
+
+const erc20ContractAddress = '0x591AD9066603f5499d12fF4bC207e2f577448c46'
+const testingAddress = '0xaaadeed4c0b861cb36f4ce006a9c90ba2e43fdc2'
+
+// 0x70a08231 = encoded "balanceOf" signature
+// aaadeed4c0b861cb36f4ce006a9c90ba2e43fdc2 = the encoded testingAddress
+// this is used in the params for eth_call
+const balanceOfQuery =
+  '0x70a08231000000000000000000000000aaadeed4c0b861cb36f4ce006a9c90ba2e43fdc2'
+let id = 1
+
+const erc20IsUp = ({ delay, maxAttempts }) => {
+  let attempts = 1
+  let lastResponse = undefined
+  return new Promise((resolve, reject) => {
+    function retrieveBalance() {
+      axios
```

Comment:
I feel like this is simple enough and the node http lib can provide what we need?